README.md

```diff
@@ -50,6 +50,105 @@ These are now documented via [Puppet Strings](https://github.com/puppetlabs/pupp
 
 For convenience, some examples are duplicated here:
 
+## rabbitmq class
```

Comment:
This should not show up once #625 is merged and I rebase

Reply:
@wyardley can you rebase? I merged #625